api/src/DTO/Invitation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\DTO;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+
+#[ApiResource(
+    collectionOperations: [],
+    itemOperations: [
+        'find' => [
+            'method' => 'GET',
+            'path' => '/{inviteKey}/find',
+        ],
+    ],
+    routePrefix: '/invitations'
+)]
+class Invitation {
+    #[ApiProperty(readable: false, writable: false, identifier: true, example: 'myInviteKey')]
+    public string $inviteKey;
+    public string $campId;
+    public string $campTitle;
+    public ?string $userDisplayName;
+    public ?bool $userAlreadyInCamp;
```

Comment:
```suggestion
    #[ApiProperty(readable: false, writable: false, identifier: true, example: 'myInviteKey')]
    public string $inviteKey;

    /**
     * The id of the camp for which this invitation is valid. This is useful for
     * redirecting the user to the correct place after they accept
     */
    #[ApiProperty(writable: false, example: '1a2b3c4d')]
    public string $campId;

    /**
     * The full title of the camp for which this invitation is valid. This should help
     * the user to decide whether to accept or reject the invitation.
     */
    #[ApiProperty(writable: false, example: 'Abteilungs-Sommerlager 2022')]
    public string $campTitle;

    /**
     * The display name of the user that is invited. May be null in case the user does
     * not already have an account.
     */
    #[ApiProperty(writable: false, example: 'Robert Baden-Powell')]
    public ?string $userDisplayName;

    /**
     * Indicates whether the logged in user is already collaborating in the camp, and
     * can therefore not accept the invitation.
     */
    #[ApiProperty(writable: false, example: true)]
    public ?bool $userAlreadyInCamp;
```

Reply:
Danke für den vorschlag, wird in den commit gesquashed.